### PR TITLE
Update build scripts to support latest boost and ubuntu distros

### PR DESCRIPTION
**Pull request by seelabs**

### CODE REVIEW

File: Builds/Ubuntu/install_rippled_depends_ubuntu.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@
 # option.
 #
 
+
```
Comment (seelabs) [inline Builds/Ubuntu/install_rippled_depends_ubuntu.sh]: fixed

**Comment by seelabs** [comment]
@ximinez I can see three options:

1) Create a script to install the latest cmake. This requires building from source, and at least on my system I used `checkinstall` to install it. I'd be mad if a script installed something in the system directories that didn't use the package manager directly. So I don't think that's a good option.

2) Don't have this script install the latest boost. Set the version to 1.61, or even 1.59 for older systems.

3) Install the latest boost (1.63). Most users will use the package manager to install boost. If they are running this script, they either need to deal with the `ABI` issue or they want the latest boost. Yes, this breaks the cmake build unless they install the latest cmake.

I vote (3); (2) seems reasonable. I don't like (1).

**Comment by mellery451** [comment]
I'll add an option 3b: install the latest boost (1.63) and then fetch the latest FindBoost.cmake from the cmake repo and install it somewhere (like /opt) where we can search for it ahead of the standard finder modules. This should only require a few lines in the cmake file to manipulate the CMAKE_MODULE_PATH. I don't know if that option is any better, but would make using boost 1.63 slightly more user friendly.

**Comment by seelabs** [comment]
In 0.60.0-b7

**Pull request closed without merging**